crates/storage/provider/src/providers/static_file/manager.rs

```diff
@@ -632,14 +632,23 @@ impl StaticFileProvider {
     where
         Provider: DBProvider + BlockReader + StageCheckpointReader + ChainSpecProvider,
     {
-        // OVM chain contains duplicate transactions, so is inconsistent by default since reth db
-        // not designed for duplicate transactions (see <https://github.com/paradigmxyz/reth/blob/v1.0.3/crates/optimism/primitives/src/bedrock_import.rs>). Undefined behaviour for queries
-        // to OVM chain is also in op-erigon.
-        if provider.chain_spec().chain() == Chain::optimism_mainnet() {
+        // OVM historical import is broken and does not work with this check. It's importing
+        // duplicated receipts resulting in having more receipts than the expected transaction
+        // range.
+        //
+        // If we detect an OVM import was done (block #1 <https://optimistic.etherscan.io/block/1>), skip it.
+        // More on [#11099](https://github.com/paradigmxyz/reth/pull/11099).
+        #[cfg(feature = "optimism")]
```

Comment:
we can keep this cfg, but not strictly required because not expensive to check and only reachable by op mainnet